duration.go

```diff
@@ -1,3 +1,8 @@
+// SPDX-License-Identifier: BSD-2-Clause
+// Provenance-includes-location: https://github.com/crewjam/saml/blob/a32b643a25a46182499b1278293e265150056d89/duration.go
+// Provenance-includes-license: BSD-2-Clause
```

Comment:
As it's still BSD-2-Clause we can probably get rid of these lines, but it's fine either way

Reply:
Let's keep it for now just in case, since it doesn't hurt :)